apps/src/music/views/MusicView.jsx

```diff
@@ -503,6 +507,16 @@ class UnconnectedMusicView extends React.Component {
       if (this.props.isProjectLevel) {
         this.analyticsReporter.onButtonClicked('play');
       }
+      // Sends a Statsig event when the Run button is pressed by a signed out user
+      // This is related to the Create Account Button A/B Test; see Jira ticket:
+      // https://codedotorg.atlassian.net/browse/ACQ-1938
+      if (this.props.signInState === SignInState.SignedOut) {
```

Comment:
@sanchitmalhotra126 I'll leave this as-is since it's a bit cleaner and already using what's in the file 😄